### ISSUE TITLE
Support alternative alias for fasd commands?

### ISSUE THREAD

**Issue by khughitt** [wookayin/fzf-fasd#5]
Greetings!

Would it be possible to extend fzf-fasd to support alternative mappings for `z`? E.g., I tend to just `j` instead of `z` as an alias to `fasd_cd -d` (habit leftover from using [autojump](https://github.com/wting/autojump) in the past..)

Thanks for taking the time to share this plugin!

Cheers.

**Comment by kakulukia** [comment]
i just noticed the same shortcoming - @khughitt did you solve that somehow over the years?

**Comment by khughitt** [comment]
@kakulukia Yep, I basically just initialize it and then redo the aliases myself.

This is [the file I source in my dotfiles](https://github.com/khughitt/dotfiles/blob/master/shell/fasd) that sets things up:

```
#
# fasd
#
export _FASD_SHELL='dash'

fasd_cache="$HOME/.fasd-init-zsh"
if [ "$(command -v fasd)" -nt "$fasd_cache" -o ! -s "$fasd_cache" ]; then
  fasd --init auto >| "$fasd_cache"
fi
source "$fasd_cache"
unset fasd_cache

# remove default aliases
unalias a
unalias s
unalias z
unalias zz

# and make some of my own
alias j='fasd_cd -d' 
alias jd="fasd_cd -d /data/"
alias jp="fasd_cd -d home d/r/"
alias js="fasd_cd -d home shiny"
alias jr="fasd_cd -d home recipe"

alias jk="cd \`fasd -d -l -r | fzf -1 --exact\`"
alias v="vim \`fasd -f -l -r | fzf -1 --exact\`"

# vi:syntax=bash
```

It includes some other things you probably don't need -- aliases to limit the searches to specific paths and such.

**Comment by kakulukia** [comment]
I already solved it forking the script and also manually activating it for my alias. But anyway thanks for your dotfiles link. That was very interesting. less is now moar - thx for that!

As it looks like you are a very heavy user of the console you might also find something interesting here: https://kakulukia.github.io/dotfiles/
